Lib/platform.py

```diff
@@ -542,6 +542,41 @@ def java_ver(release='', vendor='', vminfo=('', '', ''), osinfo=('', '', '')):
 
     return release, vendor, vminfo, osinfo
 
+
+AndroidVer = collections.namedtuple(
+    "AndroidVer", "release api_level manufacturer model device")
+
+def android_ver(release="", api_level=0, manufacturer="", model="", device=""):
+    if sys.platform == "android":
+        try:
+            from ctypes import CDLL, c_char_p, create_string_buffer
```

Comment:
If more functions are used later, it may make sense to add an _android extension to expose these functions, to avoid the dependency to ctypes, and not sensitive constants such as PROP_VALUE_MAX.